client/src/components/AdminCard/index.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Button, Pagination } from 'antd';
+import './style.css';
+
+const AdminCard = ({
+  imgUrl,
+  name,
+  description,
+  githbUrl,
+  student,
+  websiteLink,
+  studentNames,
+  ComProject,
+  CliProject,
+  editCard,
+  deleteCard,
+}) => {
+  return (
+    <>
+      <section className="card-container">
+        <ul>
+          <li>
+            <div className="name-img-card">
+              <div className="img-card">
+                <img src={imgUrl} alt="card" />
+              </div>
+              <div>
+                <h3>Name</h3>
+                <p>{name}</p>
+              </div>
+            </div>
+            {description && (
+              <div>
+                <h3>Description</h3>
+                <p>{description}</p>
+              </div>
+            )}
+            <div>
+              <h3>Github link</h3>
+              <NavLink exact to={githbUrl}>
+                Click
+              </NavLink>
+            </div>
+            {student && (
+              <div>
+                <h3>Student</h3>
+                <NavLink exact to="/admin/students">
+                  View
+                </NavLink>
+              </div>
+            )}
+            {websiteLink && (
+              <div>
+                <h3>Website</h3>
+                <NavLink exact to={websiteLink}>
+                  View
+                </NavLink>
+              </div>
+            )}
+            {studentNames && (
+              <div>
+                <h3>Student</h3>
+                <p>{studentNames}</p>
+              </div>
+            )}
+            {ComProject && (
+              <div>
+                <h3>Community p</h3>
+                <NavLink exact to="/admin/projects?type=internal">
+                  View
+                </NavLink>
+              </div>
+            )}
+            {CliProject && (
+              <div>
+                <h3>Clients p</h3>
+                <NavLink exact to="/admin/projects?type=remotely">
+                  View
+                </NavLink>
+              </div>
+            )}
+            <div>
+              <Button onClick={editCard} className="card-btn edit">
```

Comment:
When you are ready to make the edit and delete buttons functional, I think this component could handle that internally (by becoming a class component) instead of expecting that functionality to come in as click handler props.  That would make this component more encapsulated and easier to use.